packages/cloud_functions/test/cloud_functions_test.dart

```diff
@@ -38,6 +38,10 @@ void main() {
       await callable.call(<String, dynamic>{
         'quux': 'quuz',
       });
+      await CloudFunctions.instance
+          .useEmulatorFunctions(origin: 'http://localhost:5001')
```

Comment:
This still needs to be changed to useFunctionsEmulator, and is causing the test to fail.